backend/config/urls.py

```diff
@@ -16,5 +17,15 @@
         path("api-auth/", include("rest_framework.urls", namespace="rest_framework")),
     ]
 
+
+if settings.DEBUG:
```

Comment:
are these endpoints only for testing or what's the point of these?

Reply:
Yep, only for testing purposes. These will be removed at some later point.

Reply:
few `FIXME`s would make this more clear :)

Reply:
Ended up removing these endpoints already, they are useful for testing but it should also be fine to test using the correct API endpoint directly.